packages/hawtio/src/plugins/camel/icons/Icons.test.tsx

```diff
@@ -0,0 +1,22 @@
+import { render, screen, cleanup } from '@testing-library/react'
+import { IconNames, getIcon } from './Icons'
+
+describe('svg-icons', () => {
+  test('getIcon', async () => {
+    const icon1: JSX.Element = getIcon(IconNames.AggregateIcon)
+    const icon2: JSX.Element = getIcon(IconNames.GenericIcon)
+    const icon3: JSX.Element = getIcon(IconNames.RouteIcon)
+    const icon4: JSX.Element = getIcon(IconNames.EndpointsNodeIcon)
+
+    render(icon1)
+    render(icon2)
+    render(icon3)
+    render(icon4)
+
+    expect(screen.getByAltText(IconNames.AggregateIcon)).toBeInTheDocument()
+    expect(screen.getByAltText(IconNames.GenericIcon)).toBeInTheDocument()
+    expect(screen.getByAltText(IconNames.RouteIcon)).toBeInTheDocument()
+    expect(screen.getByAltText(IconNames.EndpointsNodeIcon)).toBeInTheDocument()
+    cleanup()
```

Comment:
is this cleanup necessary?  according to https://testing-library.com/docs/react-testing-library/api/#cleanup
> Please note that this is done automatically if the testing framework you're using supports the afterEach global and it is injected to your testing environment (like mocha, Jest, and Jasmine). If not, you will need to do manual cleanups after each test.